liberty-maven-plugin/src/it/dev-it/src/test/java/net/wasdev/wlp/test/dev/it/DevTest.java

```diff
@@ -217,13 +220,53 @@ public void generateFeatureTest() throws Exception {
       File helloBatchObj = new File(tempProj, "target/classes/com/demo/HelloBatch.class");
       verifyFileExists(helloBatchObj, 15000);
       // ... and run the proper mojo.
-      assertTrue(verifyLogMessageExists("Running liberty:generate-features", 10000)); // mojo ran
+      assertTrue(verifyLogMessageExists(RUNNING_GENERATE_FEATURES, 10000)); // mojo ran
```

Comment:
This may be returning true on the generate features message on startup rather than compile.  Can use the countOccurrences to ensure that is it not detecting the generate features message from startup:
```
int generateFeaturesCount = countOccurrences(RUNNING_GENERATE_FEATURES", logFile);
// Copy a Java file into place to create the HelloBatch class
...
assertTrue(verifyLogMessageExists(RUNNING_GENERATE_FEATURES, 10000, logFile, ++generateFeaturesCount))
```

Reply:
Good suggestion.